### PR TITLE
Fix missing tick in BatchNorm doc

### PR DESCRIPTION
**Pull request by ssnl**

### CODE REVIEW

**Comment by SsnL** [comment]
@googlebot I signed it!

**Review by kyscg — CHANGES_REQUESTED** [review]
Good catch, unfortunately, TensorFlow doesn't merge changes with only one typo fix etc, could you please find other such fixes, if any and make this bigger. See CONTRIBUTING.md for more details.

**Comment by SsnL** [comment]
okay @kyscg I fixed a couple more missing ticks / typos in another file.

**Comment by SsnL** [comment]
Any updates? I find it interesting that it takes less than 3 hours for someone to tell me that the patch is correct but "too small", but more than 2 weeks (and counting) for this 6-line PR to get reviewed.

**Comment by SsnL** [comment]
Nearly 50 days now.

**Comment by SsnL** [comment]
If the tf team are willing to spend 10s on this PR, as they did when this PR was first issued to tell me that this is correct but can't be merged, contributing to tf won't need to be this painful, and tf documentation won't need to be so terrible.

**Review by rthadur — APPROVED** [review]

**Comment by rthadur** [comment]
@SsnL sorry for the delay , will merge it as soon as possible.
